resources/install/linux-64/jvb.sh

```diff
@@ -15,13 +15,13 @@ if [[ "$1" == "--help"  || $# -lt 1 ]]; then
     exit 1
 fi
 
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+BASE_LIB_DIR="$(echo ~jvb)"
 
 mainClass="org.jitsi.videobridge.Main"
-cp=$(JARS=($SCRIPT_DIR/jitsi-videobridge*.jar $SCRIPT_DIR/lib/*.jar); IFS=:; echo "${JARS[*]}")
-libs="$SCRIPT_DIR/lib/native/linux-64"
-logging_config="$SCRIPT_DIR/lib/logging.properties"
-videobridge_rc="$SCRIPT_DIR/lib/videobridge.rc"
+cp="$BASE_LIB_DIR/*:$BASE_LIB_DIR/lib/*"
```

Comment:
See http://docs.oracle.com/javase/8/docs/technotes/tools/unix/classpath.html#A1100762
and specifically:

> Class path entries can contain the base name wildcard character (*), which is considered equivalent to specifying a list of all of the files in the directory with the extension .jar or .JAR